src/pages/GardenAdmin/CreateGarden.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Button, Modal } from '@mui/material'
+import { GardenConnectionForm } from 'components/GardenConnectionForm'
+import { useState } from 'react'
+import { Garden } from 'types/backend-types'
+
+const style = {
```

Comment:
Can this be given a more descriptive name? I can see at a glance that it's CSS-like. But I'd like to know what it's used for just by looking at its name, rather than searching below to see where it's passed.